src/bench/pqs/linden.cpp

```diff
@@ -35,16 +36,38 @@ Linden::~Linden()
 
 void
 Linden::insert(const uint32_t &key,
-               const uint32_t & /* Unused */)
+               const uint32_t &value)
 {
-    linden_insert(m_q->pq, key);
+    /*Add 1 to key to support 0 keys*/
+    linden_insert(m_q->pq, key+1, value);
+}
+
+void
+Linden::insert(const size_t &key,
```

Comment:
I just find it unnecessary code duplication. Wouldn't it be an option to settle on 32- or 64-bit ints for use in benchmarks? Or otherwise just use templated insert() and delete_min() methods?